libs.ts

```diff
@@ -683,24 +680,15 @@ export const libs: Lib[] = [
 		slashCommands: 'Yes',
 		buttons: 'Yes',
 		selectMenus: 'Yes',
-		threads: {
-			text: 'Dev Version',
-			url: 'https://github.com/Discord4J/Discord4J/issues/958'
-		},
+		threads: 'Yes',
```

Comment:
looks like threads are only available in the [v3.3.0-M2](https://github.com/Discord4J/Discord4J/releases/tag/3.3.0-M2) prerelease and not in the released version v3.2.6, so this change should be reverted